locale/en/docs/guides/anatomy-of-an-http-transaction.md

```diff
@@ -126,7 +126,7 @@ request.on('error', function(err) {
 });
 ```
 
-There are other ways of [handling these errors][] such as using [`domains`][] or
+There are other ways of [handling these errors][] such as
```

Comment:
@Trott how would i go about making a PR for changes that have already landed though?